docs/products/kafka/reference/kafka-metrics-prometheus.md

```diff
@@ -2,518 +2,546 @@
+| `kafka_server_DelayedOperationPurgatory_PurgatorySize_Value`                     | Size of the purgatory queue. |
+| `kafka_server_KafkaRequestHandlerPool_RequestHandlerAvgIdlePercent_OneMinuteRate` | Average idle percentage of request handlers over the last minute. |
+| `kafka_server_KafkaServer_BrokerState_Value`                                     | State of the broker. |
+| `kafka_server_ReplicaManager_IsrExpandsPerSec_Count`                             | Number of ISR expansions per second. |
+| `kafka_server_ReplicaManager_IsrShrinksPerSec_Count`                             | Number of ISR shrinks per second. |
+| `kafka_server_ReplicaManager_LeaderCount_Value`                                  | Number of leader replicas. |
+| `kafka_server_ReplicaManager_PartitionCount_Value`                               | Number of partitions. |
+| `kafka_server_ReplicaManager_UnderMinIsrPartitionCount_Value`                    | Number of partitions under the minimum ISR. |
+| `kafka_server_ReplicaManager_UnderReplicatedPartitions_Value`                    | Number of under-replicated partitions. |
+| `kafka_server_group_coordinator_metrics_group_completed_rebalance_count`         | Number of completed group rebalances. |
+| `kafka_server_group_coordinator_metrics_group_completed_rebalance_rate`          | Rate of completed group rebalances. |
+| `kafka_server_group_coordinator_metrics_offset_commit_count`                     | Number of offset commits. |
+| `kafka_server_group_coordinator_metrics_offset_commit_rate`                      | Rate of offset commits. |
+| `kafka_server_group_coordinator_metrics_offset_deletion_count`                   | Number of offset deletions. |
+| `kafka_server_group_coordinator_metrics_offset_deletion_rate`                    | Rate of offset deletions. |
+| `kafka_server_group_coordinator_metrics_offset_expiration_count`                 | Number of offset expirations. |
+| `kafka_server_group_coordinator_metrics_offset_expiration_rate`                  | Rate of offset expirations. |
+
+### Tiered storage metrics
+
+Aiven for Apache Kafka includes several metrics to monitor the performance and
+health of your Apache Kafka broker's tiered storage operations. Access these metrics
+through Prometheus to gain insights into various aspects of tiered storage, including
+data copying, fetching, deleting, and their associated lags and errors.
+
+| Metric                                                           | Description                                                                                                      |
+|------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------|
+| `kafka_server_BrokerTopicMetrics_RemoteCopyBytesPerSec_Count`    | The number of bytes per second being copied to remote storage.                                                   |
```

Comment:
Up to this point, you don't use an article (the/a/an) when starting the descriptions. I'd continue that way and wouldn't use articles as firsts words in table cells (which is mostly visual and practical thing I guess). This comment applies to all description cells in this article.

```suggestion
| `kafka_server_BrokerTopicMetrics_RemoteCopyBytesPerSec_Count`    | Number of bytes per second being copied to remote storage.                                                   |
```